### PR TITLE
Update naming convention for documentation

### PR DESCRIPTION
**Pull request by aditya-konarde**
Was: appdev-documentation
Change to: launcher-documentation

### CODE REVIEW

File: launchpad-services/launcher-documentation.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 services:
 - hash: dd46582683577a509a8be339dd59f5b754ff2171
-  name: appdev-documentation
+  name: launcher-documentation
   parameters:
-    IMAGE: registry.devshift.net/openshiftio/appdev-documentation
+    IMAGE: registry.devshift.net/openshiftio/launcher-documentation
```
Comment (gastaldi) [inline launchpad-services/launcher-documentation.yaml]: The image path should be fabric8/launcher-documentation

Comment (gastaldi) [reply launchpad-services/launcher-documentation.yaml]: ... after https://github.com/fabric8-launcher/launcher-documentation/pull/961 is merged. Before that, the image name is the same

Comment (aditya-konarde) [reply launchpad-services/launcher-documentation.yaml]: Updating.

**Comment by aditya-konarde** [comment]
@gastaldi updated.

**Comment by aditya-konarde** [comment]
Thinking we should set hash to 'None' for now and validate in Stage first

**Comment by aditya-konarde** [comment]
We can safely leave that as it is. :)

**Pull request merged**